.gitignore

```diff
@@ -42,3 +42,4 @@ pip-log.txt
 .eggs/
 .env/
 .idea/
+.vscode/
```

Comment:
That should go into _your_ `~/.gitignore`. There's nothing generating `.vscode` in this project, so it'd be better if we don't start adding everyone's favorite editor in `.gitignore`. :)

https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore